subjects/PE/PPC/hello_ppc.reko/hello_ppc_text.c

```diff
@@ -1937,8 +1937,8 @@ union Eq_n * fn00402CE0(union Eq_n * r2, struct Eq_n * r3, int32 r4, struct Eq_n
 {
 	ptr32 fp;
 	struct Eq_n * v5;
-	struct Eq_n ** tLoc28;
-	struct Eq_n * tLoc24;
+	word32 tLoc28;
+	word32 tLoc24;
```

Comment:
Seems like Reko no longer identifies `tLoc24` and `tLoc28` as pointers to things.